packages/babel-preset-fbjs/plugins/inline-requires.js

```diff
@@ -142,12 +142,29 @@ function inlineableMemberAlias(path, state) {
 }
 
 function isInlineableCall(node, state) {
-  return (
+  const isInlineable =
     node.type === 'CallExpression' &&
     node.callee.type === 'Identifier' &&
     state.inlineableCalls.hasOwnProperty(node.callee.name) &&
-    node['arguments'].length === 1 &&
+    node['arguments'].length >= 1;
```

Comment:
I guess it's fine if we also inline `require` calls with multiple arguments, can we add a simple test for this though?